source/managing_domain/01_managing_users_and_groups/users.rst

```diff
@@ -10,16 +10,23 @@ still see the same data and a subject to the same quotas as before.
 Permissions
 -----------
 
-What is changing is that you can now give them permissions to extendin what they can do. You can do so either on the
-users' index page or directly on the single user management page, both accessible through the
-**Users & groups / Users** section in your back office.
+You can now give users permissions to extend what they can do. You can do so either on the users' index page or directly on the single user management page, both accessible through the **Users & groups > Users** section in your back office.
 
-Permissions each affect an independent part of your portal so that by looking at a user's permissions list, you know
-what they can do. Each permission has its own category, symbolized by an icon and a tag color.
+Permissions each affect an independent part of your portal so that by looking at a user's permissions list, you know what they can do. Each permission has its own category, symbolized by an icon and a tag color.
 
 .. image:: images/users_permissions.png
         :alt: All available permissions, each with a category icon and color
 
+.. admonition:: Important
+   :class: important
+
+   Authenticated users can manage and generate API keys.
+   API keys inherit all permissions granted to the user for which it has been created.
+   That's why administrators should be cautious with the permissions they grant to users.
+
+   Administrators cannot revoke users' API keys.
```

Comment:
I would suggest adding "at the moment" (`Administrators cannot revoke users' API keys at the moment`) because a change is planned for the July sprint with the creation of a toolbox to revoke an API key for an administrator.